module/motion/QuinticWalk/src/QuinticWalk.cpp

```diff
@@ -219,22 +219,20 @@ namespace module::motion {
         This method computes the next motor goals and publishes them.
         */
         auto setRPY = [&](const float& roll, const float& pitch, const float& yaw) {
-            const float halfYaw   = yaw * 0.5f;
-            const float halfPitch = pitch * 0.5f;
-            const float halfRoll  = roll * 0.5f;
-            const float cosYaw    = std::cos(halfYaw);
-            const float sinYaw    = std::sin(halfYaw);
-            const float cosPitch  = std::cos(halfPitch);
-            const float sinPitch  = std::sin(halfPitch);
-            const float cosRoll   = std::cos(halfRoll);
-            const float sinRoll   = std::sin(halfRoll);
-            return Eigen::Quaternionf(cosRoll * cosPitch * cosYaw + sinRoll * sinPitch * sinYaw,  // formerly yzx
```

Comment:
I'm not actually sure I like this change, it makes things more complicated...
Maybe we just need to document what's going on whenever we use the various constructors.
Or at the very least just wrap it so we still return directly but have the vector4f in the quaternion constructor

Reply:
Goddamn this is horrible :vomiting_face: